readme.markdown

```diff
@@ -31,7 +32,7 @@ Validator | Description
 
 The `files` parameter expects an array of `File` or a `FileList`, like the ones you can pull from `input.files`, where `input` is an input element of type `file`.
 
-The files are uploaded using an `uploads` field key. Once your API has handled the uplaoded files, a JSON response should be returned. The only requirement here is a `results` property that's an array describing the success of each file upload.
+The files are uploaded using an default `uploads` field key, but this can be customized using the `fieldKey` options parameter. Once your API has handled the uploaded files, a JSON response should be returned. The only requirement here is a `results` property that's an array describing the success of each file upload.
```

Comment:
Did you want an asterisk note here, and to put the rest of that info below? Sorry, I don't quite follow.